der/src/asn1/octet_string.rs

```diff
@@ -244,13 +244,15 @@ mod bytes {
 #[cfg(test)]
 #[allow(clippy::unwrap_used)]
 mod tests {
-    use crate::asn1::{OctetStringRef, PrintableStringRef};
+    use crate::asn1::{OctetString, OctetStringRef, PrintableStringRef};
```

Comment:
I think this fails in tests because it would need to be gated behind `#[cfg(feature = "alloc")]`, but what were you trying to test?

Reply:
I'm sorry. That was just for debugging OctetString and I forgot to mark it for removal before pushing. (Normally I use `// TODO NC` for that and I have a pre-commit git hook checking it.)